CMakeLists.txt

```
@@ -35,6 +35,7 @@ include(CTest)
 # Options
 option(HOLOHUB_DOWNLOAD_DATASETS "Download datasets" ON)
 option(HOLOHUB_BUILD_PYTHON "Build Holoscan SDK Python Bindings" ON)
+option(HOLOHUB_WHEEL_TESTING "Build Applications for Python Wheel Testing" OFF)
```

Recommend that we consider support for a `HOLOHUB_BUILD_CPP_APPS` option. May need to consider how this impacts Python projects relying on C++ operator/extension builds.

Note that the `HOLOHUB_BUILD_PYTHON` option exists but might not be uniformly applied across projects right now.

Alternatively, maybe `HOLOHUB_BUILD_PYTHON_ONLY` would be clearer in explicitly disallowing C++ build paths? What do you think?

```suggestion
option(HOLOHUB_BUILD_PYTHON "Build Holoscan SDK Python Bindings" ON)
option(HOLOHUB_BUILD_CPP_APPS "Build HoloHub C++ applications" OFF)
```